src/config.ts

```diff
@@ -62,6 +62,11 @@ export const rollupChain = defineChain({
   },
   contracts: {
     ...chainConfig.contracts,
+    disputeGameFactory: {
+      [parentChain.id]: {
+        address: import.meta.env.VITE_DISPUTE_GAME_FACTORY_ADDRESS,
+      },
+    },
```

Comment:
Have you tested whether this works for chains that are not using fault proofs? Not sure if we need to make this configuration optional or not.

Reply:
Yes, I tested it on a chain that doesn't use **multi** proofs. There is no need to define these things. `DisputeGameFactory` Or you can define it as well.

**The topic I propose is mutil proof, you can test it again yourself.**
Topic: https://github.com/chaindrop-raas/oni-bridge/pull/24#issue-2623216370
cc: https://docs.optimism.io/stack/fault-proofs/explainer